docs/application/native/guides/multimedia/raw-audio.md

```diff
@@ -23,13 +23,13 @@ To play the audio PCM data, the application must call the `audio_out_create()` f
 Your application must define the following PCM data settings:
 
 - Audio channels:
-  - Mono (1 channel)
-  - Stereo (2 channels)
+  - AUDIO_CHANNEL_MONO : 1 channel
+  - AUDIO_CHANNEL_STEREO : 2 channels
 - Audio sample type:
-  - Unsigned 8-bit PCM
-  - Signed 16-bit little endian PCM
-  - Signed 24-bit little endian PCM
-  - Signed 24-bit packed in 32-bit little endian PCM
+  - AUDIO_SAMPLE_TYPE_U8 : Unsigned 8 bit integer PCM
+  - AUDIO_SAMPLE_TYPE_S16_LE : Signed 16 integer bit PCM, little endian
+  - AUDIO_SAMPLE_TYPE_S24_LE : Signed 24 bit integer PCM packed, little endian
+  - AUDIO_SAMPLE_TYPE_S24_32LE : Signed 24 bit integer PCM in LSB of 32 bit words, little endian
```

Comment:
I'm sorry for bothering you but enumeration is code.
Please use code tag for enum.
For instance, You can change like below:
```
  - `AUDIO_CHANNEL_MONO`: 1 channel 
  - `AUDIO_CHANNEL_STEREO`: 2 channels
  - ...
```

Reply:
Update done. Thanks for checking.